test/vfd_swmr_group_writer.c

```diff
@@ -4985,7 +4985,7 @@ main(int argc, char **argv)
     }
 
     /* config, tick_len, max_lag, writer, flush_raw_data, md_pages_reserved, md_file_path */
-    init_vfd_swmr_config(&config, 4, 7, writer, FALSE, 128, "./group-shadow");
+    init_vfd_swmr_config(&config, 4, 7, writer, TRUE, 128, "./group-shadow");
```

Comment:
A question, here we only test the option when the flush_raw_data is TRUE, what about when the flush_raw_data is FALSE? This option is not checked for this test. I think some other tests also haven't checked this option.

Reply:
It's up to individual test to add an option to flush/not flush raw data and then input that to init_vfd_swmr_config().  I added "not flush" to two tests here for testing: dsetops and dsetchks